docs/documentation_guidelines.md

```diff
@@ -0,0 +1,207 @@
+
+For details on organizing content and how we use Markdown, refer to [Structure and Formatting](#structure-and-formatting-guidelines).
+
+###	Use Simple English
+
+* **Be brief.** Use short sentences and paragraphs. Stick to the principle of one main idea per sentence, plus one additional point if needed. Each paragraph should address one main idea. Remember the basic structure of a paragraph: Introduction, body, and conclusion.
+* **Use simple words.** Use simple words to increase reader comprehension and reduce ambiguity. Follow our tips for making good word choices:
+   * **Avoid jargon:** Write for your audience, using everyday language where possible, and technical terms where appropriate. Avoid clichés, idioms, and metaphors.
+   * **Be consistent:** Use one term for each concept or action and use it consistently.
+   * **Avoid “fancy” words and phrases:** If there is a simpler word or phrase, use it.
+
+###	Make Content Scannable
+
+Organize your content to make it scannable for the reader, which helps them find what they need quickly, and to understand the information more efficiently.
+
+* **Put the most important content first.** Make sure your introduction clearly communicates what the reader can find on the page. Present the point of the document first, and organize supporting information towards the end of the page.
+* **Write scannable headings.** Expect readers of documentation to skim and scan the content, and to leave if they don’t find what they need quickly. Good headings add organization to your content and help the reader to find and understand content more effectively. Follow our guidelines for writing effective Headings.
+* **Write great link text.** Great link text tells the reader what they can expect when they click on a link. It also helps make the page more scannable. Follow our guidelines for writing Link text.
+
+###	Use Strong Verbs
+
+* **Passive verbs make writing stuffy and formal**. Use strong verbs to get to the point and avoid unnecessary words and phrases.
+* **Commands**, also called imperatives, are the fastest and most direct way of giving someone instructions.
+* **Use simple present tense** instead of future tense for most text. Search for the words “will” or “shall” to find future tense instances. Future tense is acceptable for conditional statements, such as in a caution or a warning.
+
+###	Parallelism
+
+Parallelism refers to the practice of using similar patterns of grammar, and sometimes length, to coordinate words, phrases, and clauses.  
+Use parallel construction in lists. The table below shows some unparallel structures and how they can be made parallel with a little rewording.
+
+| Parallel (do)           | Unparallel (don’t)        |
+|-------------------------|---------------------------|
+| 1. Mount the panel.     | 1. Mount the panel.       |
```

Comment:
The same example in both columns

Reply:
Oops, this file is here by mistake. Removed it.